packages/command-palette/src/index.tsx

```diff
@@ -91,12 +98,44 @@ export function CommandMenuGroup() {
 		setEmptyListNotice,
 		navigate,
 		currentRoute,
+		currentSiteId,
+		useSites,
 	} = useCommandPaletteContext();
-	const { commands, filterNotice, emptyListNotice } = useCommandPalette();
+	const { commands, filterNotice, emptyListNotice, singleSiteMode } = useCommandPalette();
+	const { __ } = useI18n();
+
+	const sites = useSites();
+	const currentSite = sites.find( ( site: { ID: unknown } ) => site.ID === currentSiteId );
+	const adminUrl =
+		currentSite?.options?.wpcom_admin_interface === 'wp-admin'
+			? 'https://' + currentSite.slug + '/wp-admin'
+			: 'https://wordpress.com/home/' + currentSite?.slug;
+
+	// Should just use the name but need Jetpack change for this to work in wp-admin
+	const siteName = currentSite?.name ?? currentSite?.slug;
 
 	useEffect( () => {
-		setFooterMessage( filterNotice ?? '' );
-	}, [ setFooterMessage, filterNotice ] );
+		if ( ! filterNotice && singleSiteMode ) {
+			const sitesPath = currentRoute.startsWith( '/wp-admin' )
```

Comment:
Should we make this rely the admin interface instead of the current interface?

Reply:
It's the current interface that matters - if someone prefers Calypso they can still be  on example.com/wp-admin and hence we need to send them to wordpress.com/sites.

Really this could just always be a fqdn but it's mildly annoying when you're on calypso.live and you inadvertently end up on wordpress.com.